src/main/scala/com/databricks/labs/overwatch/pipeline/Bronze.scala

```diff
@@ -95,6 +95,37 @@ class Bronze(_workspace: Workspace, _database: Database, _config: Config)
     Helpers.parClone(cloneSpecs)
 
   }
+  def snapshotStream(
```

Comment:
I think this needs to be moved to workspace and called `incrementalSnap`

currently users backup using `workspace.snap` after this is merged customers should be able to call `workspace.incrementalSnap`